packages/params/src/forkName.ts

```diff
@@ -91,5 +91,28 @@ export const forkPostElectra = exclude(forkAll, [
   ForkName.deneb,
 ]);
 export function isForkPostElectra(fork: ForkName): fork is ForkPostElectra {
-  return isForkBlobs(fork) && fork !== ForkName.deneb;
+  return isForkPostDeneb(fork) && fork !== ForkName.deneb;
 }
+
+/**
+ * Aliases only exported for backwards compatibility. This will be removed in
+ * lodestar v2.0.  The types and guards above should be used in all places as
+ * they are more correct than using the "main feature" from a fork.
+ */
+export type ForkLightClient = ForkPostAltair;
```

Comment:
minor thing but we could mark them as deprecated by adding `@deprecated` tag, I guess the comment should also not be a jsdoc attached to a single type, but just a nit

Reply:
I did that but me no likey... just sayin' it looks silly....  should i revert?  you nitty nitter 😉

Reply:
i love you nits by the way... even if I groan when i read them lol